.github/workflows/checks.yml

```diff
@@ -28,3 +28,33 @@ jobs:
       - run: yarn
       - run: yarn run prettify:diff
       - run: yarn run lint
+  mobile-tests:
+    name: Mobile Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+          check-latest: true
+      - name: Install Yarn dependencies
+        run: yarn
+      - name: Build Yarn packages
+        run: yarn build --ignore docs
+      - name: Verify locales if using l10n/* branch
+        run: |
+          if [[ "${GITHUB_REF##*/}" != l10n/* ]]; then
+            # We want to check translations only when they are complete
+            echo "Skipping translations check on non l10n branches"
+            exit 0
+          fi
+          cd packages/mobile
+          yarn --cwd packages/mobile test:verify-locales
+      - name: Run mobile tests
+        run: |
+          mkdir -p test-results/jest
+          yarn --cwd packages/mobile test:ci --runInBand
```

Comment:
We could try without `--runInBand` 
I think it was added on CircleCI because of memory issues.

Reply:
just kicked off a build without it, will see if it passes.